### PR TITLE
PFW-1497 Change Eject filament -> Eject from MMU

### PR DESCRIPTION
**Pull request by DRracer**
TODO add missing translations for:
- [x] DE
- [x] ES
- [x] HR
- [x] HU
- [x] NL
- [x] NO
- [x] RO
- [x] SK
- [x] SV

PFW-1497

### CODE REVIEW

**Comment by 3d-gussner** [comment]
@stelgenhof @leptun @Hauzman @AttilaSVK @Prime1910 @ingbrzy @shatter136 @Painkiller56 @OS-kar Please review the auto translated messages if they are correct in your language.
:+1:  correct
:-1:  needs update and please add a comment what needs to be changed

**Review by 3d-gussner — APPROVED** [review]
All looks correct and do not exceed the 16 char limit.
Hope the community translators will confirm the changes.

**Comment by AttilaSVK** [comment]
👎 the correct translation should be "Kiadás az MMU-ból", which is too long, so my suggestion is "Kiadás az MMUból" or "Kiadas az MMUbol" without accents.

**Comment by Hauzman** [comment]
Ro translation looks good to me

**Comment by stelgenhof** [comment]
The Dutch translation looks good to me.

File: lang/po/Firmware_no.po [hunk 1]
```diff
@@ -501,11 +501,11 @@ msgstr "ERR Vent på Bruker"
...
 msgstr "FEIL:"
 
-#. MSG_EJECT_FILAMENT c=17
+#. MSG_EJECT_FROM_MMU c=16
 #: ../../Firmware/messages.cpp:60 ../../Firmware/ultralcd.cpp:5111
 #: ../../Firmware/ultralcd.cpp:5549
-msgid "Eject filament"
-msgstr "Mat ut filament"
+msgid "Eject from MMU"
+msgstr "Utkast fra MMU"
```
Comment (OS-kar) [inline lang/po/Firmware_no.po]:
```suggestion
msgstr "Løs ut fra MMU"
```

**Review by OS-kar — APPROVED** [review]

File: lang/po/Firmware_no.po (hunk 1 above)
Comment (DRracer) [reply lang/po/Firmware_no.po]: Done :+1:

**Comment by Prime1910** [comment]
Croatian looks good :)

**Comment by ingbrzy** [comment]
For SK a small typo corrected

https://github.com/DRracer/Prusa-Firmware/pull/79/commits/0d3c69c41e56a88b355f64af334a494d1ade9eca

**Pull request merged**